BUILD

```diff
@@ -13,12 +13,12 @@ filegroup(
     srcs = glob([
         "test_files/**/*",
         "third_party/**/*",
+    ]) + [
         # The test suites run version-specific TypeScript compilers,
-        # but they all need the 'tslib' package to be available in
-        # a node_modules directory *above* the test/ directory.  So
-        # we must provide this extra directory as data to those tests.
-        "node_modules/tslib/*",
-    ]),
+        # but they all need the 'tslib' package to be available. So
+        # we must provide this extra filegroup as data to those tests.
+        "@npm//tslib:tslib__files",
```

Comment:
Sorry about the crazy hacks here

Reply:
s'okay, I took a stab at removing the special handling for tslib but it was pretty easy to just update the location instead